src/models/articlebookmark.js

```diff
@@ -14,7 +14,13 @@ export default (sequelize, DataTypes) => {
       articleSlug: {
         type: DataTypes.STRING,
         allowNull: false,
-        primaryKey: true
+        primaryKey: true,
```

Comment:
shouldn't this require a migration script?

Reply:
I don't think so.. 'coz we didn't change any table, we just added references.

Reply:
I think it will require migration. You are referencing another table and the relationship that is there before you add this reference will not have this relationship.

Reply:
Done